sdk/storage/azure-storage-queue/CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Release History
+
+## 12.0.0-preview.2 (2019-08-08)
+Version 12.0.0-preview.2 is a preview of our efforts in creating a client library that is developer-friendly, idiomatic to the Java ecosystem, and as consistent across different languages and platforms as possible. The principles that guide our efforts can be found in the [Azure SDK Design Guidelines for Java](https://azuresdkspecs.z5.web.core.windows.net/JavaSpec.html).
+
+For details on the Azure SDK for Java (July 2019 Preview) release, you can refer to the [release announcement](https://aka.ms/azure-sdk-preview1-java).
```

Comment:
this feels like it should be under a preview1 heading;

Reply:
Same reason as above.